spec/index.html

```diff
@@ -3632,12 +3632,15 @@ <h2>Changes since the First Public Working Draft of 24 November 2022</h2>
   <h2>Acknowledgements</h2>
 
   <p>The editors would like to thank Jeremy Carroll for his work on the
-    graph canonicalization problem, Gavin Carothers for providing valuable
-    feedback and testing input for the algorithm defined in this
-    specification, Sir Tim Berners Lee for his thoughts on graph canonicalization
-    over the years, Jesús Arias Fisteus for his work on a similar
-    algorithm.</p>
-
+    graph canonicalization problem,
+    Andy Seaborne and Gavin Carothers for providing valuable feedback and testing input
+    for the algorithm defined in this specification,
+    Sir Tim Berners-Lee for his thoughts on graph canonicalization over the years, and
+    Jesús Arias Fisteus for his work on a similar algorithm.</p>
+
+  <p>Aiden Hogan's [[Hogan-Canonical-RDF]] provided an important contemporary
+    analysis of the canonicalization problem provided an independent
+    justification of the development of <a>RDFC-1.0</a>.</p>
```

Comment:
@iherman thanks for mentioning this omission. There is an open issue on referencing this as part of the history of development, but a specific reference to the work had not yet been added..

Reply:
This paragraph appears to be an amalgam of two sentences (note the two occurrences of `provided`). I don't know enough about the particulars to provide an edit.

Reply:
Addressed; please see other comment thread.